src/test/scala/com/gu/salesforce/messageHandler/SOAPNotificationsParserTest.scala

```diff
@@ -0,0 +1,18 @@
+package com.gu.salesforce.messageHandler
+
+import com.amazonaws.util.IOUtils
+import salesforce.soap.Notifications
+import org.specs2.mutable.Specification
+
+class SOAPNotificationsParserTest extends Specification {
```

Comment:
This test is nice because you don't need to have Janus credentials to run it, unlike the `MessageHandlerLambdaTest`!